scripts/hid_configurator/configurator.py

```diff
@@ -533,23 +580,41 @@ def configurator():
     logging.info('Configuration channel for nRF52 Desktop')
 
     parser = argparse.ArgumentParser()
-    parser.add_argument('device_type', help='Selected device type: ' +
-	    ''.join("%s, " % dt for dt in DEVICE_PID)[:-2])
-    subparsers = parser.add_subparsers(dest='command')
-    subparsers.required = True
-    parser_dfu = subparsers.add_parser('dfu', help='Run DFU')
-    parser_dfu.add_argument('dfu_image', type=str, help='Path to a DFU image')
-    parser_fwinfo = subparsers.add_parser('fwinfo', help='Obtain information about FW image')
-    parser_fwreboot = subparsers.add_parser('fwreboot', help='Request FW reboot')
-    parser_config = subparsers.add_parser('config', help='Write configuration option')
-    config_subparsers = parser_config.add_subparsers(dest='module')
-    config_subparsers.required = True
-    parser_config_sensor = config_subparsers.add_parser('sensor', help='Optical sensor options')
-    parser_config_sensor_subparsers = parser_config_sensor.add_subparsers(dest='option')
-    parser_config_sensor_subparsers.required = True
-    for opt_name in SENSOR_OPTIONS.keys():
-        parser_config_sensor_opt = parser_config_sensor_subparsers.add_parser(opt_name, help=SENSOR_OPTIONS[opt_name].help)
-        parser_config_sensor_opt.add_argument('value', help='value range: {} or "fetch" to fetch the value from device'.format(SENSOR_OPTIONS[opt_name].range))
+    sp_devices = parser.add_subparsers(dest='device_type')
+    sp_devices.required = True
+    for device_name in DEVICE.keys():
```

Comment:
use `DEVICE` insted of `DEVICE.keys()` ?

Reply:
Done